arborist/server.go

```diff
@@ -114,6 +114,14 @@ func (server *Server) MakeRouter(out io.Writer) http.Handler {
 	router.Handle("/user/{username}/policy", http.HandlerFunc(server.handleUserRevokeAll)).Methods("DELETE")
 	router.Handle("/user/{username}/policy/{policyName}", http.HandlerFunc(server.handleUserRevokePolicy)).Methods("DELETE")
 
+	router.Handle("/client", http.HandlerFunc(server.handleClientList)).Methods("GET")
+	router.Handle("/client", http.HandlerFunc(parseJSON(server.handleClientCreate))).Methods("POST")
+	router.Handle("/client/{clientName}", http.HandlerFunc(server.handleClientRead)).Methods("GET")
+	router.Handle("/client/{clientName}", http.HandlerFunc(server.handleClientDelete)).Methods("DELETE")
+	router.Handle("/client/{clientName}/policy", http.HandlerFunc(parseJSON(server.handleClientGrantPolicy))).Methods("POST")
+	router.Handle("/client/{clientName}/policy", http.HandlerFunc(server.handleClientRevokeAll)).Methods("DELETE")
+	router.Handle("/client/{clientName}/policy/{policyName}", http.HandlerFunc(server.handleClientRevokePolicy)).Methods("DELETE")
```

Comment:
We probably want to rename `clientName` to `clientId`?

Reply:
either is ok, I think `clientID` since we're using that pattern for roles and policies

Reply:
OK will use that!